packages/seacas/libraries/ioss/src/CMakeLists.txt

```diff
@@ -43,8 +43,7 @@ IF (NOT ${PROJECT_NAME}_ENABLE_Zoltan)
   ADD_DEFINITIONS(-DNO_ZOLTAN_SUPPORT)
 ENDIF()
 
-IF (${PROJECT_NAME}_ENABLE_ParMETIS)
-ELSE()
+IF (NOT TPL_ENABLE_ParMETIS)
```

Comment:
@gsjaardema, as described here:

* https://tribits.org/doc/TribitsDevelopersGuide.html#how-to-add-a-new-tribits-tpl-dependency

really this should be looking at `${PACKAGE_NAME}_ENABLE_ParMETIS` stead of `TPL_ENABLE_ParMETIS`.  That allows the user to control what package enable what TPLs in case there is a problem with some package's usage of a TPL.  For example, we have used this in the past to control what packages turn on support for HDF5 (since EpetraExt HDF5 support has been problematic in the past.)

Reply:
OK, I thought on Friday you said to use `TPL_ENABLE_ParMETIS`.  Will change to use `${PACKAGE_NAME}_ENABLE_ParMETIS`

Reply:
Regarding Tests, I am also redoing this in the SEACAS repository and adding tests there.  Wanted to get this going in Trilinos so @bathmatt could test.

Reply:
> OK, I thought on Friday you said to use `TPL_ENABLE_ParMETIS`. Will change to use `${PACKAGE_NAME}_ENABLE_ParMETIS`.

@gsjaardema, setting `-DTPL_ENABLE_ParMETIS=ON` is how you turn on the ParMETIS TPL at configure time to look for ParMETIS and then turn on optional support ParMETIS in the various (SE) Packages the list ParMETIS as an optional TPL.  See:

* https://tribits.org/doc/TribitsDevelopersGuide.html#tribits-dependency-handling-behaviors

and especially:

* https://tribits.org/doc/TribitsDevelopersGuide.html#support-for-optional-se-package-tpl-is-enabled-by-default
* https://tribits.org/doc/TribitsDevelopersGuide.html#support-for-optional-se-package-tpl-can-be-explicitly-disabled

Sorry for the confusion about this.

This will be a little less confusing once TPLs and Packages are handled more consistently as per tribitspub/tribits#63.